great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -543,6 +553,14 @@ def stores(self) -> dict:
     def datasource_store(self) -> DatasourceStore:
         return self._datasource_store
 
+    @property
+    def suites(self) -> SuiteFactory:
+        if not self._suites:
```

Comment:
Unless there's a use case I don't know about, i think this confirms my suspicion that we should just raise from the constructor, or just ensure we can always instantiate the factory.